bungeecord/src/main/java/net/william278/huskchat/bungeecord/event/ChatMessageEvent.java

```diff
@@ -0,0 +1,43 @@
+package net.william278.huskchat.bungeecord.event;
+
+import net.william278.huskchat.event.IChatMessageEvent;
+import net.william278.huskchat.player.Player;
+
+public class ChatMessageEvent extends BaseEvent implements IChatMessageEvent {
+    private Player sender;
```

Comment:
Should probably @NotNull annotate a few of these fields, esp if we're exposing them as API